system/jlib/jplane.cpp

```diff
@@ -231,6 +233,39 @@ class CStoragePlane final : public CInterfaceOf<IStoragePlane>
         return hosts;
     }
 
+    virtual bool isLocalPlane() const override
+    {
+        CriticalBlock b(isLocalCrit);
+        if (cachedLocalPlane)
+            return isLocal;
+
+        isLocal = false;
+        if (hosts.size()== 0)
+        {
+            isLocal = true;
+        }
+        else
+        {
+            const char * myHostName = GetCachedHostName();
+            for (auto &host: hosts)
+            {
+                if (strsame(host.c_str(), ".") || strsame(host.c_str(), "localhost"))
+                {
+                    isLocal = true;
```

Comment:
This IStoragePlane::isLocalPlane() is really "is any part of this plane local".
I think "Plane" part of name is superflous..
don't really have a great suggestion for naming, maybe just a comment on the interface